<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceManager</class>
 <widget class="QMainWindow" name="DeviceManager">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>474</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="initGens">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Press to initialize generators...</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connectL502devices">
      <property name="text">
       <string>Press to connect L502...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="init10A">
      <property name="text">
       <string>Press to initialize 10A...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="changeSampleRate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Press to change sapmle rate...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SyncStm32">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Press to Sync with Stm32</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
